examples/diff.c

```diff
@@ -335,3 +363,27 @@ static void diff_print_stats(git_diff *diff, struct opts *o)
 	git_buf_free(&b);
 	git_diff_stats_free(stats);
 }
+
+int extract_break_rewrites(int *m, int *n, const char *arg)
+{
```

Comment:
- Ok, will add comments on the top of two `if`'s 

- Will change return 1 to return 0 and other code accordingly

- Will delete all unused items.
Is anything left, please suggest now only, because this has been pending for quite some time now.